src/main/java/io/cdap/plugin/batch/source/ftp/FTPFileSystem.java

```diff
@@ -111,7 +111,9 @@ private FTPClient connect() throws IOException {
     int port = conf.getInt("fs.ftp.host.port", FTP.DEFAULT_PORT);
     String user = conf.get("fs.ftp.user." + host);
     String password = conf.get("fs.ftp.password." + host);
+    int connectTimeout = conf.getInt(FTPConfig.FS_CONNECT_TIMEOUT, FTPConfig.DEFAULT_CONNECTION_TIMEOUT_MS);
```

Comment:
because they're used here, it's better to have these constants in this FTPFileSystem class rather than the FTPConfig class

Reply:
We are using these in config and SFTP classes also, as there was no constants class, I put it here

Reply:
right, but this config affects the behavior of the FTPFileSystem, so it should be a constant in the FTPFileSystem class, with FTPConfig referencing FTPFileSystem.CONNECT_TIMEOUT.

This is because there may be another class in the future that is unrelated to plugin FTPConfig, but that wants to configure the connect timeout for the FTPFileSystem. It makes more sense for a class like that to reference FTPFileSystem.CONNECT_TIMEOUT and not FTPConfig.CONNECT_TIMEOUT.